### DISCUSSION TITLE
Welcome to obsidian-hover-editor Discussions!

### DISCUSSION THREAD

**Discussion by nothingislost** [nothingislost/obsidian-hover-editor#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Manrunas** [comment]
Great Plugin! Just add an option to close the Hover Editor window with Escape key. Now I do it with Ctrl+w
Thank you!
